7_troubleshooting/01_common_issues.txt

```diff
@@ -0,0 +1,157 @@
+=== Some reports are in "No Answer"
+
+==== If you get no reports at all for the Node
+
+First thing to check is to see if reports were received by Rudder server.
+
+Check the last answer time (called *Last seen*) in *List Nodes* page.
+If you see:
+
+* *Never*: your Node is misconfigured or has a communication issue with the server
+* A date far (more than 15 minutes) from current time:	Synchronize server and node time
+* A recent date: check if the node has correctly updated
+
+Now we will check if promises were updated on the Node. Maybe the node could not
+update its promises anymore, even if the reporting looks ok and Rules seems to
+be applied but report keeps in ‘No Answer’.
+
+To check if a node can update its promises, run (one the node) '/var/rudder/cfengine-community/bin/cf-agent -KI | grep Update@@None'
```

Comment:
This will need changing to "rudder agent update" in 3.1+ and the grep changing too